apache/trafficcontrol
# Add a required capability to CDN in a Box's demo1 Delivery Service

## What does this PR (Pull Request) do?


- Adds the `HDD` Server Capability to CDN in a Box
- Assigns the `HDD` Server Capability to `edge`, `mid-01`, and `mid-02`
- Requires the `HDD` capability on the `demo1` Delivery Service

## Which Traffic Control components are affected by this PR?


- CDN in a Box

## What is the best way to verify this PR?

- Run CDN in a Box, make sure `https://video.demo1.ciab.test/` routes to an edge that return status code 200 after all the components have finished starting up
- Check the HDD server capability/required capability and make sure it shows up correctly

## The following criteria are ALL met by this PR

- [x] Only CDN in a Box changed, no tests necessary
- [x] Users of CDN in a Box can see the server capabilities, no documentation necessary
